test/com/ableton/VirtualEnvTest.groovy

```diff
@@ -0,0 +1,24 @@
+package com.ableton
+
+import com.lesfurets.jenkins.unit.BasePipelineTest
+import org.junit.Before
+
+import static org.junit.Assert.assertNotNull
+
+
+/**
+ * Tests for the VirtualEnv class.
+ */
+class VirtualEnvTest extends BasePipelineTest {
+  def script
+
+
```

Comment:
not sure about groovy style, but should this only be one space?

Reply:
+1

Reply:
I'm fine with this being one space (actually I would prefer it to be so). We should probably make a section in our Groovy style guide then about vertical whitespacing in that case. In any event, I'll fix this.